stratification.ts

```diff
@@ -135,7 +135,7 @@ export class StratificationGroup extends idtypes.SelectAble implements IStratifi
     });
   }
 
-  idView(idRange:ranges.Range = ranges.all()):Promise<IStratification> {
+  idView(idRange:ranges.Range = ranges.all()):Promise<any> {
```

Comment:
Not sure if there's a better solution: 
`stratification.ts(139,12): error TS2322: Type 'Promise<void>' is not assignable to type 'Promise<IStratification>'.`